### PR TITLE
Add AU article

### PR DESCRIPTION
**Pull request by deandre**
AU is proving one of those complicated TLDs. We should document some of the requirements and processes in a public-facing article.

### CODE REVIEW

File: content/articles/domains-au.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- A charity operating in Australia, as defined in the Registrant's constitution or other documents of incorporation.
+- A non-profit organization operating in Australia, as defined in the Registrant's constitution or other documents of incorporation.
+
+<warning>
+A valid Australian Business Number (ABN) is required to register a `.ORG.AU` domain name. Australian Company Numbers and Trademark Numbers are not acceptable.
+</warning>
+
+## Renewals
+
+`.AU` domains must be renewed no later than 10 days before the expiration date. Automatic and manual renewals are permitted as long as the domain is less than 90 days away from expiry. If the domain name is not renewed by the deadline, it will enter [redemption](/articles/what-happens-when-domain-expires/#redemption-period). The domain name is removed from the DNS and cannot be updated. This period lasts for 31 days; during this period, the domain name can be recovered for an additional fee.
```
Comment (weppos) [inline content/articles/domains-au.markdown:64]: > Automatic and manual renewals are permitted as long as the domain is less than 90 days away from expiry.

Unfortunately, we are limited by our current partner and we can't support manual renewal.

Comment (deandre) [reply content/articles/domains-au.markdown:64]: @weppos wait, these tlds are auto renew only?

Comment (weppos) [reply content/articles/domains-au.markdown:64]: At the time being yes, .AU is auto-renewal only. You cannot manually renew it.

Comment (deandre) [reply content/articles/domains-au.markdown:64]: Ahh, I see. Should the suffixes be updated? The name tool indicates otherwise.

Comment (weppos) [reply content/articles/domains-au.markdown:64]: After review, my comment is no longer valid.

**Review by itsalyse — APPROVED** [review]

**Review by SarahCaminiti1 — APPROVED** [review]

**Pull request merged**